tidb-cloud/ticloud-connect.md

```diff
@@ -0,0 +1,59 @@
+---
+title: ticloud connect
+summary: The reference of `ticloud connect`.
+---
+
+# ticloud connect
+
+Connect to a TiDB Cloud cluster:
+
+```shell
+ticloud connect [flags]
+```
+
+> **Note:**
+>
+> The connection forces the ANSI SQL mode for the session.
+You can get more information about the ANSI SQL mode from [ANSI SQL mode](https://dev.mysql.com/doc/refman/8.0/en/sql-mode.html#sqlmode_ansi)
+
+## Examples
+
+Connect to a TiDB Cloud cluster in interactive mode:
+
+```shell
+ticloud connect
+```
+
+Use the default user to connect to a TiDB Cloud cluster in non-interactive mode:
+
+```shell
+ticloud connect -p <project-id> -c <cluster-id>
+```
+
+Use a specific user to connect to the TiDB Cloud cluster in non-interactive mode:
```

Comment:
Do customers need to create the specific user in advance?

Reply:
Yes